docs/setup.md

```diff
@@ -12,93 +12,138 @@
 
 The following steps will prepare you to run complete analyses of SARS-CoV-2 data by installing required software and running a simple example workflow.
 
-## 1. Make a copy of this tutorial
+## 1. Setup your Nextstrain environment with Conda
 
-There are two ways to do this:
+The following instructions use [Conda](https://docs.conda.io/en/latest/) to install the tools you'll need for this tutorial.
+Conda is a package and environment management system that allows you to install Python and other software into controlled environments without disrupting other software you have installed (e.g., on your computer, your shared cluster, etc.).
 
-  * [Recommended] If you're familiar with git, clone this repository either via the web interface, a GUI such as [GitKraken](https://www.gitkraken.com/), or the command line:
+<p style="color: #055160; background-color: #cff4fc; border-color: #b6effb; padding: 1em; border-radius: .25rem;">
+If you use Microsoft Windows, [install the Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+Follow [instructions to open a new WSL window for your Linux distribution](https://docs.microsoft.com/en-us/windows/wsl/wsl-config) and then run the following commands.
+</p>
+
+### Create the Nextstrain environment
+
+[Install Miniconda with Python 3 for your operating system](https://docs.conda.io/en/latest/miniconda.html), update Conda to the latest version, and install Mamba for faster installation.
 
 ```bash
-git clone https://github.com/nextstrain/ncov.git
+conda update -n base conda
+conda install -n base -c conda-forge mamba
 ```
 
-  * [Alternative] If you're not familiar with git, you can also download a copy of these files via the buttons on the left.
+Create a Conda environment named ``nextstrain``.
+This command will install [Git](https://git-scm.com/), [Snakemake](https://snakemake.readthedocs.io/en/stable/), and [Nextstrain's full toolkit](https://docs.nextstrain.org/en/latest/install-nextstrain.html), the tools you'll need to work through this tutorial.
 
-## 2. Setup your Nextstrain environment
+```bash
+mamba create -n nextstrain -c conda-forge -c bioconda \
+  augur auspice nextstrain-cli nextalign snakemake awscli git pip
```

Comment:
Are all of these dependencies really necessary if the user decides to run the CLI with the `--use-conda` flag? (Tangentially, should a conda "runner" be added to the CLI?)

Reply:
No, these aren't all strictly necessary. I opted to use the same environment here as the core Nextstrain installation docs to avoid confusion, but only Augur, the CLI, Snakemake, and Git are currently required.

We have discussed the idea of a Conda runner for the CLI or at least letting the CLI manage the Conda environment. This would allow us to run something like `nextstrain update` to update the Conda environment or the Docker image, depending on how the CLI has been configured.